pipeline.jl

```diff
@@ -260,12 +266,22 @@ flush(stdout);
         )
         fname = joinpath(outdir, "apred/$(mjd)/" * outfname * ".h5")
         safe_jldsave(fname, metadata; dimage, ivarimage, chisqimage, CRimage, saturation_image)
+        if save3dcal
```

Comment:
are you trying to determine how many issues are being introduced in the 3D cals? This should just be the 1/f and amplifier offsets (and eventually crosstalk). Also, it looks like you are saving the uncalibrated data as "cal"... but maybe I am misreading.